truffle.js

```diff
@@ -58,8 +67,8 @@ module.exports = {
       network_id: "*",
       host: "localhost",
       port: 8545,
```

Comment:
not necessary anymore - passed in custom provider

Reply:
done